src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  font-family: inherit;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-weight: 300;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input_small {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.labelSearchBar::before {
+  content: "";
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  background-image: url(./images/Search.svg);
+}
+
+.label_small::before {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+  background-size: 11px;
+}
+
+:root {
+  --color: #3d4e61;
+}
```

Comment:
better to place :root selector on top of the file

Reply:
Okay, I`ll replace